dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -563,10 +564,16 @@ private void updateInfo(Workflow oldWorkflow, Workflow newWorkflow) {
         List<Token> tokens = tokenDAO.findZenodoByUserId(user.getId());
         if (!tokens.isEmpty()) {
             Token zenodoToken = tokens.get(0);
-            String refreshUrl = zenodoUrl + "/oauth/token";
-            String payload = "client_id=" + zenodoClientID + "&client_secret=" + zenodoClientSecret
-                    + "&grant_type=refresh_token&refresh_token=" + zenodoToken.getRefreshToken();
-            refreshToken(refreshUrl, zenodoToken, client, tokenDAO, null, null, payload);
+
+            // Check that token is an hour old
```

Comment:
Also, I wonder if this should be configurable. Probably overkill, but if Zenodo ever changes their access token expiry...

Reply:
This would be worth investigating for all tokens we store.